build-system/common/check-package-manager.js

```diff
@@ -310,8 +327,8 @@ function checkPythonVersion() {
   // Python2 prints its version to stderr (fixed in Python 3.4)
   // See: https://bugs.python.org/issue18338
   const pythonVersionResult =
-    getStderr(`${pythonExecutable} --version`).trim() ||
-    getStdout(`${pythonExecutable} --version`).trim();
+    getStderr('python --version').trim() ||
+    getStdout('python --version').trim();
```

Comment:
`getStdout('python --version 2>&1').trim()` redirects stderr to stdout, rather than running this twice

Reply:
Would using this shorthand break things on Windows?